src/presenters/membrePresenter.ts

```diff
@@ -0,0 +1,233 @@
+import { formaterEnDateFrancaise } from './shared/date'
+import { HistoriqueViewModel } from './shared/historique'
+import { formatMontant } from './shared/number'
+import { RoleViewModel, toRoleViewModel } from './shared/role'
+
+export function membrePresenter(codeDepartement: string): MembreViewModel {
+  return {
+    aidantsEtMediateurs: {
+      liste: [
+        {
+          fonction: ['Coordinateur', 'Médiateur numérique'].join(', '),
+          lienFiche: '/',
+          logos: [],
+          nom: 'Martin Dubois',
+        },
+        {
+          fonction: ['Médiateur numérique', 'Conseiller numérique', 'Aidant numérique', 'Aidants Connect'].join(', '),
+          lienFiche: '/',
+          logos: [`${process.env.NEXT_PUBLIC_HOST}/conum.svg`, `${process.env.NEXT_PUBLIC_HOST}/aidant-numerique.svg`],
+          nom: 'Marc-Olivier Gagnon',
+        },
+        {
+          fonction: ['Médiateur numérique', 'Aidant numérique'].join(', '),
+          lienFiche: '/',
+          logos: [`${process.env.NEXT_PUBLIC_HOST}/mednum.svg`],
+          nom: 'Philippe Morin',
+        },
+      ],
+      totalAidant: 6,
+      totalCoordinateur: 1,
+      totalMediateur: 4,
+    },
+    contactReferent: {
+      email: 'didier.durand@example.com',
+      fonction: 'Directeur',
+      nom: 'Durant',
+      prenom: 'Didier',
+      telephone: '0102030405',
+    },
+    conventionsEtFinancements: {
+      conventions: [
+        {
+          documents: [
+            {
+              conseilleNumerique: 'Paul Durant',
+              contratDeTravail: 'CDD',
+              expiration: formaterEnDateFrancaise(new Date('2025-09-23')),
+              libelle: 'Contrat de travail 220925',
+              lien: '/',
+              lienConseilleNumerique: '/',
+              statut: {
+                libelle: 'En cours',
+                variant: 'success',
+              },
+              telechargement: '/',
+            },
+            {
+              conseilleNumerique: 'Paul Durand',
+              contratDeTravail: 'CDD',
+              expiration: formaterEnDateFrancaise(new Date('2025-09-23')),
```

Comment:
```suggestion
              expiration: formaterEnDateFrancaise(new Date('2024-09-23')),
```

comme ligne 65 :  `libelle: 'Expirée',`